snapshot/snapshot.go

```diff
@@ -403,8 +403,9 @@ func (o *snapshotter) Remove(ctx context.Context, key string) error {
 		return errors.Wrap(err, "failed to remove")
 	}
 	var cleanupBlob bool
-	if _, ok := snap.Labels[label.NydusDataLayer]; ok {
-		cleanupBlob = true
+	blobDigestString, cleanupBlob := snap.Labels[label.NydusDataLayer]
```

Comment:
No type is required in a variable name, just use `blobDigest`.

Reply:
fixed